### ISSUE TITLE
[BUG] Search-Advanced click error in toolbar

### ISSUE THREAD

**Issue by mansurdegirmenci** [vitmalina/w2ui#2006]
![1](https://user-images.githubusercontent.com/38755096/116992191-52fd9600-acde-11eb-87c0-c8aae33a64f6.PNG)

![2](https://user-images.githubusercontent.com/38755096/116992199-55f88680-acde-11eb-9ad5-dabc2961e361.PNG)

**Comment by mpf82** [comment]
Thanks for your bug report.

Closing issue: fixed in current master.

**Comment by mansurdegirmenci** [comment]
![Animation](https://user-images.githubusercontent.com/38755096/119490078-136a1d00-bd65-11eb-9f81-22acfae07099.gif)

:)

**Comment by mpf82** [comment]
@mansurdegirmenci  thanks, I'll prepare a PR

**Comment by mansurdegirmenci** [comment]
![Animation](https://user-images.githubusercontent.com/38755096/119538806-25af7f80-bd94-11eb-9069-488159b56646.gif)

**Comment by mpf82** [comment]
@mansurdegirmenci Can you please create a new issue, add some description to it and add the relevant code part as text? It's hard to follow the GIF, because you cannot pause it.
